python/lsst/ts/standardscripts/auxtel/integration_tests/dometrajectory_mcs.py

```diff
@@ -251,7 +251,12 @@ def show_tel_elaz(data):
             f"GoToPosition={AzimuthCommandedState.GOTOPOSITION}"
         )
         assert (
-            abs(subtract_angles(dome_pos.azimuthPosition, self.track_elaz[1])) <= 0.1
+            abs(
+                salobj.angle_diff(dome_pos.azimuthPosition, self.track_elaz[1])
+                .to(u.deg)
+                .value
```

Comment:
Please consider spelling this as `salobj.angle_diff(...).deg`. It's much simpler -- here and elsewhere.